(self.webpackChunk_N_E=self.webpackChunk_N_E||[]).push([[229],{4309:function(t,e,n){(window.__NEXT_P=window.__NEXT_P||[]).push(["/home",function(){return n(7084)}])},4138:function(t,e,n){"use strict";function r(t,e){(null==e||e>t.length)&&(e=t.length);for(var n=0,r=new Array(e);n<e;n++)r[n]=t[n];return r}function i(t,e){return function(t){if(Array.isArray(t))return t}(t)||function(t){if("undefined"!==typeof Symbol&&null!=t[Symbol.iterator]||null!=t["@@iterator"])return Array.from(t)}(t)||function(t,e){if(t){if("string"===typeof t)return r(t,e);var n=Object.prototype.toString.call(t).slice(8,-1);return"Object"===n&&t.constructor&&(n=t.constructor.name),"Map"===n||"Set"===n?Array.from(n):"Arguments"===n||/^(?:Ui|I)nt(?:8|16|32)(?:Clamped)?Array$/.test(n)?r(t,e):void 0}}(t,e)||function(){throw new TypeError("Invalid attempt to destructure non-iterable instance.\\nIn order to be iterable, non-array objects must have a [Symbol.iterator]() method.")}()}n.d(e,{U:function(){return p}});var o=n(7297),a=n(5893),s=n(7294),c=n(8123),u=n(4144),l=n(7357);function d(){var t=(0,o.Z)(["\n  cursor: pointer;\n"]);return d=function(){return t},t}var f=n(6829).Z.img(d()),h={position:"absolute",top:"50%",left:"50%",maxHeight:"90vh",maxWidth:"1400px",width:"85vw",transform:"translate(-50%, -50%)",bgcolor:"background.paper",border:"2px solid #000",boxShadow:24,p:4},p=function(){var t=i(s.useState(!1),2),e=t[0],n=t[1];return(0,a.jsxs)("div",{style:{display:"flex"},children:[(0,a.jsx)(c.l,{}),(0,a.jsxs)("div",{style:{maxWidth:"1200px",display:"block",marginLeft:"20px"},children:[(0,a.jsx)("h3",{children:"Identification of protein substrates of the Cullin-RING E3 ubiquitin ligase CRL4 through its substrate recognition factors DCAFs"}),"Lab Web Site :"," ",(0,a.jsx)("a",{href:"http://fmboisvert.recherche.usherbrooke.ca/equipe/",children:"http://fmboisvert.recherche.usherbrooke.ca/equipe/"}),(0,a.jsx)(u.Z,{open:e,onClose:function(){return n(!1)},children:(0,a.jsx)(l.Z,{sx:h,children:(0,a.jsx)("img",{alt:"Visual Abstract",src:"https://raw.githubusercontent.com/laboFMB/DCAF-data/main/abstract.png",width:"100%",height:"auto"})})}),(0,a.jsx)(f,{onClick:function(){return n(!0)},alt:"Visual Abstract",src:"https://raw.githubusercontent.com/laboFMB/DCAF-data/main/abstract.png"}),(0,a.jsx)("h5",{children:"Cullin-RING finger ligases (CRLs) represent the largest family of ubiquitin ligases and are responsible for ubiquitination of ~20% of cellular proteins degraded through the proteasome, catalyzing the transfer of E2-loaded ubiquitin to a substrate. Seven Cullins were described in vertebrates and among them, CUL4 associates with DDB1 as adaptor to form the CUL4-DDB1 ubiquitin ligase complex, involved in protein ubiquitination and regulation of many cellular processes. The specificity of CUL4-DDB1 is mediated by substrate recognition adaptors named DDB1/CUL4 associated factors (DCAF), which are characterized by the presence of a short structural motif of approximately 40 amino acids terminating in a tryptophan (W)-aspartic acid (D) dipeptide, the WD40 domain. Moreover, the majority of WDR40 protein shown to interact with DDB1 present WDXR motif. Using different approaches (bioinformatics/structural analyses), independent studies identified at least 60 different WD40 containing proteins that could act as adaptors for the DDB1/CUL4 complex. The aim of the project is to identify the protein partners of each DCAF using affinity purification followed by mass spectrometry and confirm their association with DDB1-CUL4. The DDB1 interactome was defined by overexpressing DDB1-GFP and DDB1-BirA proteins and identification of interactors after GFP/biotin IPs. Each DCAFs were fused to BirA protein and proximity labeling of potential interactors was performed to identify individual DCAFs interactome. Because identification of protein substrates for E3-ubiquitin ligases is not always guaranteed by identifying protein interactions, we measured changes in protein stability or degradation to identify proteins targeted by each DCAFs. In conclusion, this work provides new insights into the roles of DCAFs in DDB1-CUL4 complex, protein targeting, and cellular process affected. link to the paper, etc."})]})]})}},8123:function(t,e,n){"use strict";n.d(e,{l:function(){return b}});var r=n(7297),i=n(5893),o=(n(7294),n(6829)),a=n(1664),s=n.n(a),c=n(1163),u=n(6638),l=n(4682);function d(){var t=(0,r.Z)(["\n  width: 50px;\n  height: 100vh;\n  background-color: black;\n"]);return d=function(){return t},t}function f(){var t=(0,r.Z)(["\n  &:hover {\n    cursor: pointer;\n    background-color: # !important;\n    color: black !important;\n  }\n\n  ","\n"]);return f=function(){return t},t}var h=o.Z.div(d()),p=o.Z.div(f(),(function(t){return t.active&&"\nbackground-color: #04AA6D !important;\ncolor: white !important;\n  "})),b=function(){var t=(0,c.useRouter)();return(0,i.jsxs)(h,{children:[(0,i.jsx)(p,{active:"/"===t.pathname,children:(0,i.jsx)(s(),{href:"/",children:(0,i.jsx)(u.Z,{sx:{color:"white",padding:"8px"},fontSize:"large"})})}),(0,i.jsx)(p,{active:"/data"===t.pathname,children:(0,i.jsx)(s(),{href:"/data",children:(0,i.jsx)(l.Z,{sx:{color:"white",padding:"8px"},fontSize:"large"})})})]})}},7084:function(t,e,n){"use strict";n.r(e);var r=n(4138);e.default=r.U}},function(t){t.O(0,[206,774,888,179],(function(){return e=4309,t(t.s=e);var e}));var e=t.O();_N_E=e}]);